lib/surface/compiler.ex

```diff
@@ -878,7 +878,8 @@ defmodule Surface.Compiler do
   defp validate_tag_children([]), do: :ok
 
   defp validate_tag_children([%AST.SlotEntry{name: name} | _]) do
-    {:error, "slot entries are only allowed as children elements of components, but found slot entry for #{name}"}
+    {:error,
+     "Slot entries are only allowed as children elements of components. No slot found named slot. Did you mean <##{name} />"}
```

Comment:
I think this should only be the message if it's the default slot

Reply:
I add other named slots and tried to ensure this only happen with the default slot.

Reply:
good catch, I add a new function for named slots, not just only default ones, and it raises the same error message when using `:` instead of `#`